### ISSUE TITLE
Workflow for QGIS, QGIS Server, and QWC2?

### ISSUE THREAD

**Issue by d3netxer** [qgis/qwc2-demo-app#70]
does anybody have a good guide on how to start using QGIS, QGIS Server, and QWC2?

There are good presentations online on the great features and capabilities of these components but I have a hard time finding guides on how to use them in practice.

![img](https://user-images.githubusercontent.com/2665840/34414940-08838a96-ebba-11e7-813d-db871bbe322d.png)

I would be willing to help with documentation.

So to recap I am interested in getting from QGIS desktop and creating a QGIS project, then taking that project and uploading to QGIS server. Then QGIS server would be able to display the project using QWC2 on the Internet.

**Comment by d3netxer** [comment]
I was able to save a QGIS project file and upload it to my server that has QGIS Server installed. The QGIS server is working. Now I am having trouble figuring out how QGIS Web Client is supposed to work.

**Comment by manisandro** [comment]
I've added some documentation which should be pretty complete: https://github.com/qgis/qwc2-demo-app/blob/master/doc/QWC2_Documentation.md

**Comment by andreasneumann** [comment]
Thank you - much appreciated!
